src/Summoner/CLI.hs

```diff
@@ -1,7 +1,8 @@
-{-# LANGUAGE ApplicativeDo   #-}
-{-# LANGUAGE QuasiQuotes     #-}
-{-# LANGUAGE TemplateHaskell #-}
-{-# LANGUAGE TupleSections   #-}
+{-# LANGUAGE ApplicativeDo    #-}
+{-# LANGUAGE QuasiQuotes      #-}
+{-# LANGUAGE TemplateHaskell  #-}
+{-# LANGUAGE TupleSections    #-}
+{-# LANGUAGE TypeApplications #-}
```

Comment:
You can remove `TypeApplications` from this module since it's now in `default-extensions`

Reply:
If it is removed, parser error returns in lines where it uses.

Reply:
Can you show us which error you are facing?

Reply:
![default](https://user-images.githubusercontent.com/28530747/44574270-4e992500-a7a2-11e8-82b7-082e0df01239.png)

Reply:
Well, it's not the compilation error. It's `hlint` error. You need to modify your `hlint` file to say there about this extension. It shouldn't affect the library code.